node/constants.js

```diff
@@ -29,6 +29,7 @@ module.exports = {
   DC_DOWNLOAD_FAILURE: 20,
   DC_DOWNLOAD_IN_PROGRESS: 1000,
   DC_DOWNLOAD_UNDECIPHERABLE: 30,
+  DC_EVENT_BACKGROUND_FETCH_COMPLETED_FOR_ALL_ACCOUNTS: 2200,
```

Comment:
this must be same as in rust / ffi:

```suggestion
  DC_EVENT_ACCOUNTS_BACKGROUND_FETCH_DONE: 2200,
```